test/devFunds.js

```diff
@@ -31,7 +31,8 @@ describe('embark.DevFunds', function () {
     account: {
       password: './test/test1/password',
       numAccounts: 3,
-      balance: "5 ether"
+      // this conversion is normally done when constructing a Config object
+      balance: utils.getWeiBalanceFromString("5 ether", Web3)
```

Comment:
@danlipert I'm not sure I follow the comment up there? Why exactly are we converting it here in the config (that should actually receive the human readable format)?

I'm sure I'm missing something...

Reply:
It's my understanding that this is a mock config object used here for the test rather than a real config object that would be loaded and processed from a configuration file. Since it doesn't go through the `lib/core/config.js` code, and thus get converted, I convert it manually here. This may make this test obsolete however - I do test the conversion of this field in the blockchain configuration tests that I added in an earlier commit with my changes in this commit to `test/config.js` and `test/test1/config/blockchain.json`.

Reply:
Also I may need some assistance with the rebase - I took a stab at it but got caught in a bunch of manual merges. I'm thinking I may be able to squash all of my changes into one commit and then cherry-pick that on the new `develop` branch. Let me know what you think the best strategy is.

Reply:
> Since it doesn't go through the lib/core/config.js code, and thus get converted, I convert it manually here.

Ah, I didn't have that on my radar. Thanks for clarifying!

Reply:
re:rebase: Yes, probably easiest to just squash them and rebase the squashed one on top of latest `develop`.

I can also take care of that if you feel uncomfortable doing that.